Code/V2/src/components/key.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../themes/index';
+
+const Key = (props) => {
+        var keyColor; 
+        switch (props.status) {
+                case 'flashed':
+                        keyColor = theme.key.flashed;
+                        break;
+                case 'selected':
+                        keyColor = theme.key.selected;
+                        break;
+                default:
+                        keyColor = theme.key.neutral;
+
+        }
+
+        const Button = styled.button`
+        font-size: 1em;
+        color: ${theme.key.script};
+        margin: 0.25em;
+        padding: 0.25 0.5em;
+        border-radius: 8px;   // border-radius seems to round the button's edges with larger values and sharpen with smaller
+        background: ${keyColor};   
+        border: 2px solid ${keyColor};
```

Comment:
If the border is the same colour as the background, it likely does not have any change. Is there any reason you have chosen to include this? Do you believe we can remove the "border" styling all together?

Reply:
The default border color seemed to be black when I was running the app. I'm looking back at the styled-components docs and somehow the border color is the same as the background color, but for me somehow it's not.